apps/backend/api/ai/gpt_migration.py

```
@@ -77,42 +77,58 @@ class ModelCapabilities:
         latency_ms_p50=2000,
         deprecation_date=datetime(2025, 7, 14),  # CRITICAL DEADLINE
```

Ruff (0.14.4)

`datetime.datetime()` called without a `tzinfo` argument

(DTZ001)